src/transcript/TranscriptResult.ts

```diff
@@ -2,11 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import TranscriptAlternative from './TranscriptAlternative';
+import TranscriptLanguageWithScore from './TranscriptLanguageWithScore';
 
 export default class TranscriptResult {
   resultId: string;
   channelId?: string;
   isPartial: boolean;
+  languageCode?: string;
```

Comment:
what is this for? Is this the language code with most confidence score from `languageIdentifications` ? Do we have a documentation in general to understand what all these properties mean?

Reply:
`languageCode` in `TranscriptResult` is not a required field, hence made it optional.

Details can be found in https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_Result.html

Reply:
Yes. While there is no written literature that `languageCode` would be the language code with the most confidence score from `languageIdentifications`. 

Transcribe's documentation states that it will be the most dominant language in provided transcribe stream - https://docs.aws.amazon.com/transcribe/latest/dg/lang-id-stream.html

In my testing, I have always found `languageCode` the one with the highest score (confidence)